lib/gemstash/cli/start.rb

```diff
@@ -36,6 +37,12 @@ def store_daemonized
         Gemstash::Env.daemonized = daemonize?
       end
 
+      def check_rubygems_version
+        STDERR.puts("Rubygems version is too old, please update rubygems by running: " \
+                    "gem update --system") unless
+                    Gem::Requirement.new("~> 2.4").satisfied_by?(Gem::Version.new(Gem::VERSION))
```

Comment:
Also, do you think it makes sense to colorize the message as red or yellow to draw more attention to it, or is that overkill?